core/src/components/modal/test/card/modal.e2e.ts

```diff
@@ -105,4 +105,112 @@ test.describe('card modal', () => {
       expect(content).toHaveJSProperty('scrollY', true);
     });
   });
+  test.describe('card modal: rendering - tablet', () => {
```

Comment:
Would it be possible to bring these tests into a helper method so we don't have so much repeated code? Ditto for the other tests where tablet versions have been added, but especially this one since it has so many `test` calls and may have more added later.

Reply:
Sure. What parts are you thinking we should move to helper functions?

Reply:
For `basic/modal.e2e.ts` and `custom/modal.e2e.ts` I created a function that both the mobile and tablet visual tests execute in the body of the `test` callback.

For `card/modal.e2e.ts` I ended up creating a Page Object Model (https://playwright.dev/docs/test-pom) to remove a good chunk of boilerplate.

There's still some duplication, but it's not as excessive as before.

Reply:
Muuuuch better, thank you!